app/src/main/java/org/mozilla/fenix/settings/creditcards/interactor/CreditCardsManagementInteractor.kt

```diff
@@ -19,6 +19,8 @@ interface CreditCardsManagementInteractor {
      * @param creditCard The selected [CreditCard] to edit.
      */
     fun onSelectCreditCard(creditCard: CreditCard)
+
+    fun onAddCreditCard()
```

Comment:
We can add the comment above here. I would use some variation of the `onSelectCreditCard ` kDoc.